### ISSUE TITLE
The order of topnav items is different for each language

### ISSUE THREAD

**Issue by jihoon-seo** [cncf/glossary#880]
I just noticed that the the order of topnav items is different for each language.
I guess that those items are displayed in the alphabetical order of each language.

[en]: the items are in the English alphabetical order.
![image](https://user-images.githubusercontent.com/46767780/167782141-851d281f-3c08-4b87-ad38-23daa73234ff.png)

[ko]: the items are in the Korean alphabetical order.
![image](https://user-images.githubusercontent.com/46767780/167782432-d81fc913-a7ce-4bc4-883a-18e1af604ab6.png)

[zh]
![image](https://user-images.githubusercontent.com/46767780/167782624-72c12b99-cbe2-4a53-9900-1d1757a9ab11.png)

and so on..

---

If we decide to align the order of those items over all languages,
we might try to use [the Hugo feature of "weight"](https://gohugo.io/templates/lists/#by-weight) for each page.

**Comment by seokho-son** [comment]
@jihoon-seo Thanks for suggestion. 
However, I am not sure it is worth to rearrange. 
Is it important? To me, alphabetical ordering seems ok.
But, why not, we can apply your suggestion. :)

**Comment by cjyabraham** [comment]
Yes. All three pages have weight set to 10 so I guess they are just being listed alphabetically. It's an easy change if people would prefer them in a particular order.

**Comment by seokho-son** [comment]
Hi @jihoon-seo 
Do you want to apply and work on this issue?

**Comment by jihoon-seo** [comment]
Sorry that I missed the last comment. I am okay & happy to close this issue as not planned.
